caproto/curio/client.py

```diff
@@ -237,7 +238,8 @@ def __init__(self, *, log_level='ERROR'):
                                     [ca.RepeaterConfirmResponse(
                                         repeater_address='127.0.0.1')]
                                     )
-                        await self.broadcaster.command_queue.put(response)
+                        await self.command_queue.put(
```

Comment:
Wrong queue, and the item to be put for the broadcaster would be (addr, [commands])